source/includes/fact-connect-srv-description.rst

```diff
@@ -0,0 +1,28 @@
+*New in Compass version 1.12*
+     
+This toggle indicates whether the provided :guilabel:`Hostname`
+is an
+:manual:`SRV Record </reference/connection-string/#dns-seedlist-connection-format>`.
+
+SRV connection strings have a prefix of ``mongodb+srv:``.
+If you are using an SRV connection string, you do not need to
+include ``mongodb+srv`` in your :guilabel:`Hostname`.
+
+.. example::
+
+    If this is the SRV connection string for your MongoDB instance:
+
+    .. code-block:: shell
+        :copyable: false
+
+        mongodb+srv://<username>:<password>@gettingstarted-7q2cs.mongodb.net/test
+
+    In |compass-short|, you would specify your
+    :guilabel:`Hostname` as:
+
+    .. code-block:: shell
+        :copyable: false
+
+        gettingstarted-7q2cs.mongodb.net
+
+If this toggle is enabled, you do not need to specify a :guilabel:`Port`.
```

Comment:
I think this sentence should be the second sentence in this section, and the "This toggle indicates whether..." sentence.

Reply:
I wonder if "Port" needs to be titlecased and bolded since we are referring to just a 'port' and not the field "Port". Thoughts?

Reply:
Excellent - just noticed I had a typo in my first comment but you still understood me :)